sw/device/tests/aes_sideload_test.c

```diff
@@ -79,10 +109,28 @@ void aes_test(void) {
   CHECK_DIF_OK(dif_aes_read_output(&aes, &out_data));
   CHECK_ARRAYS_NE(out_data.data, kPlainText, ARRAYSIZE(kPlainText));
 
+  // Finish the ECB encryption transaction.
+  LOG_INFO("Finished encrypting with AES sideloaded key");
   // Finish the ECB encryption transaction.
   CHECK_DIF_OK(dif_aes_end(&aes));
 
-  LOG_INFO("Decrypting the cypher text with the side loaded key.");
+  if (kDeviceType == kDeviceSimDV) {
+    LOG_INFO("AES Plaintext (HW Core): 0x%08x%08x%08x%08x",
+             in_data_plain.data[0], in_data_plain.data[1],
+             in_data_plain.data[2], in_data_plain.data[3]);
+    LOG_INFO("AES Ciphertext (HW Core): 0x%08x%08x%08x%08x", out_data.data[3],
+             out_data.data[2], out_data.data[1], out_data.data[0]);
+    LOG_INFO(
+        "AES Expected Ciphertext (from C model): "
+        "0x%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x%02x",
+        kAESDigest[15], kAESDigest[14], kAESDigest[13], kAESDigest[12],
+        kAESDigest[11], kAESDigest[10], kAESDigest[9], kAESDigest[8],
+        kAESDigest[7], kAESDigest[6], kAESDigest[5], kAESDigest[4],
+        kAESDigest[3], kAESDigest[2], kAESDigest[1], kAESDigest[0]);
+    CHECK_ARRAYS_EQ(out_data.data, (uint32_t *)kAESDigest, 4);
```

Comment:
why 4? isn't it 16?
```suggestion
    CHECK_ARRAYS_EQ(out_data.data, (uint32_t *)kAESDigest, sizeof(out_data.data));
```

Reply:
It's the size of the unit32_t array, but I fixed it anyway to be able to use to sizeof